app/components/register/register.controller.js

```diff
@@ -0,0 +1,29 @@
+;(function () {
+  angular
+    .module("todo")
+    .controller("RegisterController", RegisterController);
+
+  RegisterController.$inject = ["$scope", "$http", "$state", "API"];
+
+  function RegisterController($scope, $http, $state, API) {
+    $scope.signUp = signUp;
+    this.registerSucces = registerSucces;
+    this.registerError = registerError;
+
+
+    function signUp() {
+      $http.post(API.BASE + API.REGISTRATION, $scope.user)
+        .then(registerSucces, registerError);
+    }
+
+
+    function registerSucces(response) {
+      $state.go("groups");
```

Comment:
look at https://github.com/NikitaSedyx/todo-dj/blob/login1.0/items/resources.py
after registration session won't be opened, fix it here or in the backend